.gitignore

```diff
@@ -17,3 +17,8 @@ node_modules
 *.pyc
 notebooks/reply-views/basic.conf
 *.env
+python/notebooks/stream-seeding/data
```

Comment:
This shouldn't be changing in this commit. This PR should only add changes to `blob` and `public/ss`.